pom.xml

```diff
@@ -42,21 +43,17 @@
             <groupId>org.apache.kafka</groupId>
             <artifactId>connect-api</artifactId>
             <version>${kafka.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka-clients</artifactId>
-            <version>${kafka.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.azure</groupId>
             <artifactId>azure-cosmos</artifactId>
-            <version>4.11.0</version>
+            <version>4.14.0</version>
         </dependency>
         <dependency>
-            <groupId>io.confluent</groupId>
-            <artifactId>kafka-schema-registry-maven-plugin</artifactId>
-            <version>6.0.1</version>
+            <groupId>com.azure</groupId>
```

Comment:
`azure-cosmos` already brings `azure-core-http-netty` - why do we need this explicitly ?

Reply:
maybe he's bringing in a specific version of netty?

Reply:
Yes, with the latest `azure-cosmos` , we transitively bring in `azure-core-http-netty`, which transitively pulls in `netty:4.1.60.Final`, which contains a critical CVE. I had to pull this in separately, pinned to a newer version to workaround.

![Screen Shot 2021-05-13 at 7 51 22 PM](https://user-images.githubusercontent.com/816633/118214496-aed5c680-b424-11eb-8f21-fa1c3b19b26f.png)

Reply:
@ncliang - we recently released azure-cosmos 4.15.0 - which transitively brings netty version 4.1.63.Final. Let's upgrade azure-cosmos to 4.15.0

Reply:
> we transitively bring in `azure-core-http-netty`, which transitively pulls in **`netty:4.1.60.Final`, which contains a critical CVE**

@kushagraThapar i believe that what @ncliang  is saying is that netty 4.1.63.Final has issues with it, so you should update the cosmos client to use a later version of netty (which he does here by explicitly setting the version he wants)

Reply:
@ryancrawcour now really, @ncliang is talking about version 4.1.60.Final and not 4.1.63.Final. 
And azure-cosmos version 4.15.0 actually depends on azure-core-http-netty version 1.9.2 - please refer here - https://mvnrepository.com/artifact/com.azure/azure-cosmos/4.15.0

@ncliang - please update azure-cosmos to 4.15.0 in this PR, and remove the azure-core-http-netty dependency from pom.xml

Reply:
+1, as there are different netty component, including boring-ssl, and their versions need to match, I would just rely on Cosmos DB to bring its dependency. Please don't override.

Reply:
@ncliang could you please take a look at the suggestion from @kushagraThapar